### PR TITLE
[CHAD-8785] Changed heating and cooling range for Aeotec thermostat.

### PR DESCRIPTION
**Pull request by Bhavik-T**
Changed heating and cooling range for Aeotec thermostat from 0-40°C to 8-28°C.

### CODE REVIEW

**Review by greens — APPROVED** [review]

File: drivers/SmartThings/zwave-thermostat/profiles/base-thermostat-8to20C.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+components:
+- id: main
+  capabilities:
+  - id: temperatureMeasurement
+    version: 1
+  - id: thermostatHeatingSetpoint
+    version: 1
+    configuration:
+          - values:
+            key: "temperature.value"
```
Comment (bflorian) [inline drivers/SmartThings/zwave-thermostat/profiles/base-thermostat-8to20C.yml:11]: This needs to be `heatingSetpoint.value`. The capability attribute name is `heatingSetpoint`, not `temperature`

File: drivers/SmartThings/zwave-thermostat/profiles/base-thermostat-8to20C.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            range: [8, 20]
+  - id: thermostatCoolingSetpoint
+    version: 1
+    configuration:
+          - values:
+            key: "temperature.value"
```
Comment (bflorian) [inline drivers/SmartThings/zwave-thermostat/profiles/base-thermostat-8to20C.yml:17]: This needs to be `coolingSetpoint.value`. The capability attribute name is `coolingSetpoint `, not `temperature`

**Review by bflorian — CHANGES_REQUESTED** [review]
OK except for the attribute name issues noted.

**Comment by iplantemn** [comment]
@greens @bflorian updated capability configuration

**Comment by lelandblue** [comment]
We need to change "configuration:" to "config:" please take a look @ctowns, per @iplantemn

**Comment by lelandblue** [comment]
Hello @ctowns "configuration:" to "config:" please?

**Pull request merged**